### PR TITLE
Prune new stopwords from database

### PR DESCRIPTION
**Pull request by peterrauscher**

### CODE REVIEW

File: .github/workflows/stopwords-change.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    runs-on: ubuntu-latest
+    needs: build_and_push_image
+    steps:
+      - name: Deploy to Digital Ocean droplet via SSH action
+        uses: appleboy/ssh-action@v0.1.3
+        with:
+          host: ${{ secrets.HOST }}
+          username: ${{ secrets.USERNAME }}
+          key: ${{ secrets.SSHKEY }}
+          passphrase: ${{ secrets.PASSPHRASE }}
```
Comment (justinoboyle) [inline .github/workflows/stopwords-change.yml:46]: need to be careful with logs here

File: .github/workflows/stopwords-change.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          envs: IMAGE_NAME,REGISTRY,{{ secrets.DIGITALOCEAN_ACCESS_TOKEN }},GITHUB_SHA
+          script: |
+            # Login to registry
+            docker login -u ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }} -p ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }} registry.digitalocean.com
```
Comment (justinoboyle) [inline .github/workflows/stopwords-change.yml:50]: this will work but needs configuration of the `.env`s -- until we put the DB in the dockerfile

**Review by justinoboyle — CHANGES_REQUESTED** [review]
can we do stopwords PR separately from new deployment workflow?

File: .github/workflows/stopwords-change.yml (hunk 1 above)
Comment (peterrauscher) [reply .github/workflows/stopwords-change.yml:46]: BTW, Github automatically censors secrets in logs/terminal output w/ Actions. Not relevant anymore but learned that yesterday working on separate project and thought it was interesting.

**Pull request closed without merging**